### DISCUSSION TITLE
Neolink is not a recognized command.

### DISCUSSION THREAD

**Discussion by dflatimer** [thirtythreeforty/neolink#375] [Q&A]
Hello, when i attempt to run  neolink rtsp --config config.toml from the directory where i installed the files it tells me that neolink is not a recognized command (im summarizing).  Im not a coder or an expert, but just curious how you all are doing it.  TYIA.  Running windows.

**Comment by versile2** [comment]
open command prompt. odds are you are in powershell. If you want to run it from powershell you need to specify neolink.exe and sometimes it will make you do .\neolink.exe

Limited information but that's where I'd start.
